blog/2024-12-05-community-days.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Join Us for the Exciting Third Eclipse Tractus-X Community Days!
```

Comment:
```suggestion
title: Join Us for the Third Eclipse Tractus-X Community Days!
```

just as a comment, I'd keep it precise and it's of course exciting, but IMO no need to say that in the title.